### ISSUE TITLE
Stacks: bulk delete non-primary photos 

### ISSUE THREAD

**Issue by discosilvia** [photoprism/photoprism#1904]
Hi all,

The stacking feature succesfully identifies similar/identical photos from my collection. 

Is there any way to delete non-primary photos from a stack **in bulk**. I know it is possible to go through them manually, check the stack and delete unwanted duplicates, but a more automatic way of selecting stacks would be a useful feature.

Thx!

**Comment by lastzero** [comment]
You can multi select and delete full stacks. Why would you bulk delete non primary images without having them seen before? If you view them, it's just one click to delete them.

**Comment by leofidus** [comment]
If you go through a stack of near-identical photos to select the best one as primary, you then have to go through again to delete all the other ones. A button to "resolve" a stack by bulk deleting the non-primary photos would make that workflow much quicker.

**Comment by discosilvia** [comment]
Yes that’s exactly it! The workflow proposed by @leofidus would solve my issue.
